pkg/shares/parse_test.go

```diff
@@ -220,13 +220,14 @@ func generateRawShare(namespace appns.Namespace, isSequenceStart bool, sequenceL
 	rawShare = append(rawShare, byte(infoByte))
 	rawShare = append(rawShare, sequenceLenBuf...)
 
-	return padWithRandomBytes(rawShare)
+	return padWithRandomBytes(t, rawShare)
 }
 
-func padWithRandomBytes(partialShare []byte) (paddedShare []byte) {
+func padWithRandomBytes(t *testing.T, partialShare []byte) (paddedShare []byte) {
 	paddedShare = make([]byte, appconsts.ShareSize)
 	copy(paddedShare, partialShare)
-	rand.Read(paddedShare[len(partialShare):])
+	_, err := rand.Read(paddedShare[len(partialShare):])
+	assert.NoError(t, err)
```

Comment:
nit: I'd generally `require` no error here. This is bootstrapping code. There is no point in continuing if this errors for some reason

Reply:
resolved by https://github.com/celestiaorg/celestia-app/pull/1692/commits/7155cb503ff9677571c759fee0ecaa92f3dd2974